src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -866,22 +883,16 @@ public void downloadParticipantRoster(String appId, String userId, String passwo
         String requestJson = jsonObjectMapper.writeValueAsString(requestMsg);
 
         // sent to SQS
-        String queueUrl = bridgeConfig.getProperty(CONFIG_KEY_DOWNLOAD_ROSTER_SQS_URL); //TODO ask about the url
+        String queueUrl = bridgeConfig.getProperty(CONFIG_KEY_DOWNLOAD_ROSTER_SQS_URL);
         SendMessageResult sqsResult = sqsClient.sendMessage(queueUrl, requestJson);
         LOG.info("Sent request to SQS for userId=" + userId + ", app=" + appId +
                 "; receipted message ID=" + sqsResult.getMessageId());
     }
 
-    private void checkPassword(String password) {
-        String regex = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,}$";
-
-        Pattern pattern = Pattern.compile(regex);
-        Matcher matcher = pattern.matcher(password);
-
-        if (!matcher.matches()) {
-            throw new BadRequestException("Password must contain at least 8 characters: " +
-                    "1 upper case letter, 1 lower case letter, and 1 number.");
-        }
+    private void validatePassword(String password) {
+        Errors error = Validate.getErrorsFor(password);
+        PasswordPolicy passwordPolicy = new PasswordPolicy(8, true, false, true, true);
+        ValidatorUtils.validatePassword(error, passwordPolicy, password);
     }
```

Comment:
You should move this to ParticipantRosterRequestValidator. ValidatorUtils.validatePassword() won't throw if there's an error.